libs/ui-components/package.json

```diff
@@ -36,10 +47,10 @@
     "@patternfly/react-icons": "^5.0.0",
     "@patternfly/react-styles": "^5.1.2",
     "@patternfly/react-table": "^5.0.0",
-    "i18next": "^21.8.14",
-    "react": "17.0.1",
-    "react-dom": "17.0.1",
-    "react-i18next": "^11.7.3",
+    "i18next": ">=21.8.14",
```

Comment:
Should these versions be perhaps more limited?
What versions are we targeting?

Reply:
i18next 23.15.2
react 18.3.1
react-dom 18.3.1
react-i18next 15.0.2

I've updated the ranged. Let me know if you want them restricted even more.

Reply:
Looks good now, thanks